test-requirements.txt

```diff
@@ -1 +1,2 @@
 git+https://github.com/nerc-project/coldfront-plugin-cloud@main#egg=coldfront_plugin_cloud
+git+https://github.com/15five/django-scim2@33e7caa
```

Comment:
This is not necessary. You have already defined `django-scim2` as a dependency in `requirements.txt`.